docs/src/guides/tables.md

```diff
@@ -284,6 +294,35 @@ This guide will show how to create tables, insert data into them, and update the
         "price": 20.0
     }]);
     ```
+    ### From an Arrow Table
```

Comment:
These are two different sections?
Can we put them into the same Section "From an Arrow Table", but two different tabs, as what happens in the quickstart.

Also we can include the code in real ts file and make sure they compile / tests

For example, https://github.com/lancedb/lancedb/blob/main/docs/src/basic_legacy.ts